test/Dialect/FIRRTL/infer-widths-errors.mlir

```diff
@@ -157,27 +157,6 @@ firrtl.circuit "Foo" {
   }
 }
 
-// -----
-// https://github.com/llvm/circt/issues/5002
-
-firrtl.circuit "Issue5002" {
-  // expected-error @below {{uninferred width: port "ref" cannot satisfy all width requirements}}
-  firrtl.module private @InRef(in %ref : !firrtl.rwprobe<uint>) { }
-  firrtl.module @Issue5002(in %x : !firrtl.uint<1>, in %y : !firrtl.uint<2>) {
-    %w1, %w1_ref = firrtl.wire forceable : !firrtl.uint, !firrtl.rwprobe<uint>
-    %w2, %w2_ref = firrtl.wire forceable : !firrtl.uint, !firrtl.rwprobe<uint>
-    firrtl.connect %w1, %x : !firrtl.uint, !firrtl.uint<1>
-    firrtl.connect %w2, %y : !firrtl.uint, !firrtl.uint<2>
-
-    %inst1_ref = firrtl.instance inst1 @InRef(in ref: !firrtl.rwprobe<uint>)
-    %inst2_ref = firrtl.instance inst2 @InRef(in ref: !firrtl.rwprobe<uint>)
-    firrtl.ref.define %inst1_ref, %w1_ref : !firrtl.rwprobe<uint>
-    // expected-note @below {{width is constrained to be at most 1 here:}}
-    // expected-note @below {{width is constrained to be at least 2 here:}}
-    firrtl.ref.define %inst2_ref, %w2_ref : !firrtl.rwprobe<uint>
-  }
-}
```

Comment:
We need multiple drivers for the same static (in the IR, like a specific wire or port).  Since probe's must be driven exactly once this is a bit tricky to test.

An input probe driven in different instantiation contexts with different widths is clearest way I know to do this (this is test I had commented out earlier for same "seems like a good thing to have under test still" thinking).
( https://github.com/llvm/circt/pull/6921#discussion_r1594774391 )

I had made a comment previously about instance-choice (if has an output probe of unknown width, in theory the two choices might try to infer to different widths based on their contents), but I'm not sure story there re:unknown widths and interaction with inferences.

So, not sure how to keep this as a test but that would be nice, agreed, thanks!